CDTS Blazor/Pages/IdentityDocuments.razor

```diff
@@ -32,17 +32,75 @@
         </div>
     </form>
 
+    @if (@loading == true)
+    {
+        // TODO : Add a loading animation instead of just text.
+        <div>Loading.....</div>
+    }
+
+    // TODO : Make the alerts closeable
+    // TODO : Keep file names so the user knows that the files haven't disapeared
```

Comment:
These are currently showing on the page. Move these comments 2 lines down (below the curly brace but above the div).